src/style.css

```
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.productCard {
+  width: 200px;
+  padding: 14px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.productCard:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.productCard-imgBox {
```

BEM
```suggestion
.product-card__img-box {
```

if there is `product-card` block